cli/template/extras/config/_eslintrc.cjs

```diff
@@ -19,7 +19,13 @@ const config = {
     project: path.join(__dirname, "tsconfig.json"),
   },
   plugins: ["@typescript-eslint"],
-  extends: ["next/core-web-vitals", "plugin:@typescript-eslint/recommended"],
+  extends: [
+    "next/core-web-vitals",
+    "plugin:@typescript-eslint/recommended",
+    "eslint-config-next",
+    "eslint-config-prettier",
+    "prettier",
```

Comment:
Doesnt this cause conflicts?

Reply:
yeah you might be right. leme start a discussion on the parent pr as im not sure what we are going to go for with the extended eslint config